.github/workflows/tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
         gdal-version: ["3.4.0", "3.3.3"]
         include:
           - python-version: "3.8"
```

Comment:
Nice! Makes sense to me we test all the python envs only on github. Downside is we won't catch issues with this locally and it will be hard to reproduce, but I think it's alrigth

Reply:
I think we can direct people to use tox locally for local testing, and we currently direct people to run github actions locally with `act` in the CONTRIBUTING.MD: https://github.com/CosmiQ/solaris/blob/0.5.0/CONTRIBUTING.md

I think act is probably the way to go so probably the CONTRIBUTING.md does not need to be changed based on the changes in this PR.

Reply:
I tried running tox locally using act.

I had to switch to a smaller container from osgeo to make it work across different python & gdal versions.